test/Test.hs

```diff
@@ -470,17 +470,17 @@ testTransaction :: Test
 testTransaction = testCase "transaction" $ do
     watch ["{1}k1", "{1}k2"] >>=? Ok
     unwatch            >>=? Ok
-    set "foo" "foo" >>= \case
+    set "{1}k1" "foo" >>= \case
       Left _ -> error "error"
       _ -> return ()
-    set "bar" "bar" >>= \case
+    set "{1}k2" "bar" >>= \case
       Left _ -> error "error"
       _ -> return ()
-    foobar <- multiExec $ do
-        foo <- get "foo"
-        bar <- get "bar"
-        return $ (,) <$> foo <*> bar
-    assert $ foobar == TxSuccess (Just "foo", Just "bar")
+    k1k2 <- multiExec $ do
```

Comment:
@stoeffel for context: in the wip implementation on the cluster branch there's a `multiExecWithHash` function that does work in cluster mode. It takes an additional key argument that is used to determine which hashslot to send the initial `MULTI` command too (`MULTI` doesn't have a key itself, so we wouldn't know where to send it otherwise).

I think we should find a way to make regular `multiExec` work for clusters as well, probably by holding off sending transaction commands until we get one with a KEY, at which point we know where to send everything.